src/main/scala/mesosphere/marathon/api/v2/PodsResource.scala

```diff
@@ -31,4 +50,207 @@ class PodsResource @Inject() (
   def capability(@Context req: HttpServletRequest): Response = authenticated(req) { _ =>
     ok()
   }
+
+  // TODO(jdef) force parameter? probably since this supports deployment; but what does it actually
+  // mean to "force" a create?
+  @POST @Timed
+  def create(
+    body: Array[Byte],
+    @DefaultValue("false")@QueryParam("force") force: Boolean,
+    @Context req: HttpServletRequest): Response = authenticated(req) { implicit identity =>
+
+    withValid(unmarshalJson[PodDef](decodeBytes(body, req))) { podDef =>
+
+      val pod = PodDefinition(withDefaults(podDef, podDefaults)).withCanonizedIds()
+
+      withAuthorization(CreateRunSpec, pod) {
+        val (createdPod, deploymentId) = podSystem.create(pod, force)
+        Events.maybePost(PodEvent(req.getRemoteAddr, req.getRequestURI, PodEvent.Created))
+
+        val builder = Response
+          .created(new URI(createdPod.id.toString))
+          .entity(marshalJson(createdPod.asPodDef))
+
+        deploymentId.foreach(id => builder.header(DEPLOYMENT_ID_HEADER, id))
+        builder.build()
+      }
+    }(createPodValidator)
+  }
+
+  @GET @Timed
+  def findAll(@Context req: HttpServletRequest): Response = authenticated(req) { implicit identity =>
```

Comment:
Is the information you get here always the same?
In `v2/apps` we use embed parameter to customize the response.